src/controllers/organizationController.ts

```diff
@@ -65,8 +72,52 @@ export default class OrganizationController {
       }
       return withValidatedOrValidationError(
         OrganizationRegistrationParams.decode(req.body),
-        (organizationRegistrationParams: OrganizationRegistrationParams) =>
-          registerOrganization(organizationRegistrationParams, user)
+        async (
+          organizationRegistrationParams: OrganizationRegistrationParams
+        ) => {
+          const errorResponseOrSuccessResponse = await registerOrganization(
+            organizationRegistrationParams,
+            user
+          );
+          return errorResponseOrSuccessResponse.map(async response => {
+            const organization = response.payload;
+            const outputFolder = `./documents/${organization.ipa_code}`;
+            fs.mkdirSync(outputFolder, { recursive: true });
```

Comment:
since this code runs into UI layer, you must avoid using synchronous methods that will block the UI for _all_ the connected users. I'm very skeptical anyway to put this kind of expensive functions (sending email, generate pdf, ...) into the UI code. this should (must ?) be handled by offline asynchronous workers (eventually, using a shared queue)